controllers/auth.controllers.js

```diff
@@ -0,0 +1,26 @@
+const admin = require("firebase-admin");
+
+// Initialize the Firebase Admin SDK
+const serviceAccount = require("../nodeassign-firebase-adminsdk-7rp5j-0a456fe229.json");
+admin.initializeApp({
```

Comment:
move this initialization code in fireabse-config.js file @mukeshsin

Reply:
ok sir @meanstack9716